tools/cldr-code/src/test/java/org/unicode/cldr/test/TestEn001Time.java

```diff
@@ -0,0 +1,34 @@
+package org.unicode.cldr.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+import org.unicode.cldr.util.*;
+
+public class TestEn001Time {
+
+    static final String TIME_FORMAT_PREFIX =
+            "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/timeFormats/timeFormatLength";
```

Comment:
Note that the ticket refers to "standard time formats". I'm not sure whether that applies to any calendars other than Gregorian; currently only Gregorian is checked. Also, I'm not sure whether all paths starting with this prefix qualify as "standard". This part of the test currently passes. The test could be revised later if and when the need arises to add paths that would fail.

Reply:
Currently, en_001.xml contains no paths starting with "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/timeFormats/timeFormatLength".

Reply:
Yes, and we want to keep it that way.